src/java.base/share/classes/java/io/CharArrayWriter.java

```diff
@@ -184,8 +185,9 @@ public CharArrayWriter append(CharSequence csq) {
      * {@code csq} is not {@code null}, behaves in
      * exactly the same way as the invocation
      *
-     * <pre>
-     *     out.write(csq.subSequence(start, end).toString()) </pre>
+     * {@snippet lang=java :
+     *     out.write(csq.subSequence(start, end).toString())
```

Comment:
a trailing ";' would be useful for copy/paste; butr that's not the existing style.  Your call.

Reply:
These statements in general do not have a trailing `;`; left unchanged.